src/js/components/Search.js

```diff
@@ -163,7 +163,10 @@ export default class Search extends Component {
   }
 
   _onInputKeyDown (event) {
```

Comment:
I believe we need that i both places. if drop is active and we dont select anything and press enter the callback is not being invoked.